tsconfig.json

```diff
@@ -0,0 +1,44 @@
+{
+  "compilerOptions": {
+    "target": "ES2022",
+    "lib": ["ES2022"],
+    "module": "CommonJS",
+    "skipLibCheck": true,
+    "outDir": "./build",
+
+    /* Bundler mode */
+    "moduleResolution": "Node",
+    // "allowImportingTsExtensions": true,
+    "esModuleInterop": true,
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+
+    "incremental": true,
+    "declaration": true,
+    "strict": true,
+
+    "declarationMap": true,
+		"sourceMap": true,
```

Comment:
wrong indentation

Reply:
I think it is a bug in the web version. 
Within the editor I see the correct indentation

Reply:
Are used tabs instead of spaces

Reply:
```suggestion
    "sourceMap": true,
```